Wrappers/Python/test/test_CIL_CVXPy.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
+#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   substantial contributions by UKRI-STFC and University of Manchester.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from cil.optimisation.functions import L2NormSquared
+from cil.optimisation.functions import TotalVariation
+
+from cil.utilities import dataexample
+from cvxpy import *
```

Comment:
Can you us `import cvxpy`, then use `cvxpy.Minimise` etc. as the call. This makes it clearer to read the tests as otherwise you don't know where 'Variable' and 'Minimise' come from.

Reply:
fixed